packages/quantic/cypress/integration/breadcrumb-manager/breadcrumb-manager-expectations.ts

```diff
@@ -0,0 +1,111 @@
+import {should} from '../common-selectors';
+import {SearchExpectations} from '../search-expectations';
+import {
+  BreadcrumbManagerSelector,
+  BreadcrumbManagerSelectors,
+  BreadcrumbSelector,
+} from './breadcrumb-manager-selectors';
+
+function baseBreadcrumbManagerExpectations(
+  selector: BreadcrumbSelector,
+  name: string
+) {
+  return {
+    displayBreadcrumb: (display: boolean) => {
+      selector
+        .get()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the ${name} breadcrumb`);
+    },
+    displayLabel: (display: boolean) => {
+      selector
+        .label()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(
+          `${should(display)} display the label of ${name} breadcrumb`
+        );
+    },
+    labelContains: (label: string) => {
+      selector
+        .label()
+        .contains(label)
+        .logDetail(`should have the label "${label}"`);
+    },
+    displayValues: (display: boolean) => {
+      selector
+        .values()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(
+          `${should(display)} display the values of ${name} breadcrumb`
+        );
+    },
+    numberOfValues: (value: number) => {
+      selector
+        .values()
+        .should('have.length', value)
+        .logDetail(`should display ${value} ${name} breadcrumb values`);
+    },
+    breadcrumbValueLabelAtIndexContains: (value: string) => {
+      selector
+        .breadcrumbValueLabelAtIndex(0)
+        .contains(value)
+        .logDetail(`should have the value "${value}" at first value`);
+    },
+    displayShowMore: (display: boolean) => {
+      selector
+        .showMoreButton()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(
+          `${should(display)} display the show more button ${name} breadcrumb`
+        );
+    },
+  };
+}
+
+function breadcrumbManagerExpectations(selector: BreadcrumbManagerSelector) {
+  return {
+    displayBreadcrumbManager: (display: boolean) => {
+      selector
+        .get()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the breadcrumb manager`);
+    },
+    displayClearFilters: (display: boolean) => {
+      selector
+        .clearFilters()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the breadcrumb manager`);
+    },
+  };
+}
+
+export const BreadcrumbManagerExpectations = {
+  ...breadcrumbManagerExpectations(BreadcrumbManagerSelectors),
+  facet: {
```

Comment:
is this the facet or the facetBreadcrumb? if it's the latter then I recommend all the names be updated to add precision.

Reply:
I change it to `facetBreadcrumb`